.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,12 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Could we `.gitignore` the `.idea` directory? Would prefer not to include individual IDE prefs in the project.

Reply:
Good catch! Forgot to check that.

Reply:
Done.